sdk/python/kfp/v2/google/client/schedule.py

```diff
@@ -97,9 +97,10 @@ def create_from_pipeline_file(
   pipeline_jobs_api_url = f'https://{_CAIPP_DEFAULT_ENDPOINT}/{_CAIPP_API_VERSION}/projects/{project_id}/locations/{region}/pipelineJobs'
 
   # Preparing the request body for the Cloud Function processing
-  full_pipeline_name = pipeline_dict.get('name')
-  pipeline_display_name = pipeline_dict.get('displayName')
+  pipeline_id = pipeline_dict['pipelineSpec']['pipelineInfo']['name']
   time_format_suffix = "-{{$.scheduledTime.strftime('%Y-%m-%d-%H-%M-%S')}}"
+  pipeline_display_name = f"{pipeline_id}{time_format_suffix}"
+  full_pipeline_name = 'projects/{}/pipelineJobs/{}'.format(project_id, pipeline_display_name)
   if 'name' in pipeline_dict:
```

Comment:
It looks like this attribute is not used in the subsequent process. Is it really necessary?